CHANGELOG.md

```diff
@@ -6,7 +6,10 @@
 
 ##### Enhancements
 
-* None.
+* `VariableNameRule` now accepts symbols starting with more than one uppercase
+  letter to allow for names like XMLString or MIMEType.
```

Comment:
Do those 2 trailing spaces have any purpose? They are removed by every markdown formatter. Also they are TRAILING spaces. Exactly the thing you don't want.